lib/plugins/aws/package/compile/events/alb/lib/targetGroups.js

```diff
@@ -62,7 +68,12 @@ module.exports = {
               Id: resolveLambdaTarget(functionName, functionObj),
             },
           ],
-          Name: this.provider.naming.getAlbTargetGroupName(functionName, albId, multiValueHeaders),
+          Name: this.provider.naming.getAlbTargetGroupName(
```

Comment:
Technically it could be: `targetGroupName || this.provider.naming.getAlbTargetGroupName(..)`.

Reply:
Indeed it felt a bit strange to pass it as a parameter but with the naming used I thought it would be better to handle the naming in only one place. What do you think ?

Maybe should I rename it to something like `generateAlbTargetGroupName` ? As you prefer

Reply:
> Maybe should I rename it to something like generateAlbTargetGroupName ? As you prefer

I had exactly same thought. We're a bit cautious with renames as it may happen that some external plugin may attach to that (even though this area is considered private), but in this case it's unlikely, so I think we can rename